hyperspy/misc/image_tools.py

```diff
@@ -218,3 +218,38 @@ def contrast_stretching(data, saturated_pixels):
     vmin = np.percentile(data, saturated_pixels / 2.)
     vmax = np.percentile(data, 100 - saturated_pixels / 2.)
     return vmin, vmax
+
+MPL_DIVERGING_COLORMAPS = [
+    "BrBG",
+    "bwr",
+    "coolwarm",
+    "PiYG",
+    "PRGn",
+    "PuOr",
+    "RdBu",
+    "RdGy",
+    "RdYIBu",
+    "RdYIGn",
+    "seismic", ]
```

Comment:
according to [this](http://matplotlib.org/examples/color/colormaps_reference.html), `Spectral` is missing... Though it's possible you skipped it intentionally?